index.js

```diff
@@ -245,7 +245,11 @@ class Mapeo extends events.EventEmitter {
   }
 
   close (cb) {
-    this.sync.close(cb)
+    this.osm.core.pause(() => {
+      this.osm.core._logs.close(() => {
+        this.sync.close(cb)
```

Comment:
in desktop/mobile we do sync.close first, is there a difference?

Reply:
If there are any replications in progress, then I think closing the logs will cause an error. But `sync.close` will also cause an error in the replication so 🤷‍♂️ But maybe it's best to ensure that nothing is writing to the logs before we close them?